test/compilable/imports/test15925.d

```diff
@@ -0,0 +1,9 @@
+module imports.test15925;
+
+public mixin template AddImports ()
+{
+    import core.thread;
```

Comment:
Although `core` modules are more stable than Phobos, it would be better to remove dependency to minimize tests.

Add one more file `imports/test15925thread.d` and declare `alias void delegate(void*, void*) nothrow ScanAllThreadsFn;` in it.